### PR TITLE
Add page 13 and hep_b vaccine rules

### PR DESCRIPTION
**Pull request by labhinav**

### CODE REVIEW

**Comment by labhinav** [comment]
@harshitgupta412 please review

File: claims_analysis/cardinal_care/cardinal_care.hrf [hunk 1]
```diff
@@ -331,8 +399,8 @@ valid_service_provider(outpatient_surgical_services, surgical_assistant)
 
 valid_screening_list([obesity,healthy_diet,alcohol_misuse,tobacco_missuse,depression,sti,
 genetic_risk_cancer,stress_management,chronic_condition,lactation,prenatal_postpartum]).
-covered_list_with_location([physician_consultation,allergy,inpatient_care,non_surgical_physican_service,inpatient_surgical_services])
```
[the remark this answers is not in the document]
Comment (labhinav) [reply claims_analysis/cardinal_care/cardinal_care.hrf:334]: fixed

File: claims_analysis/cardinal_care/cardinal_care.hrf [hunk 2]
```diff
@@ -36,6 +36,74 @@ valid_hospitalization(C,P):-
...
+minor_or_dependent(C,Cl):-
+  person.dependent(Cl,yes).
+
+minor_or_dependent(C,Cl):-
+  person.dob(Cl,DOB) &
+  claim.time(C,C_D,C_T) &
+  get_age(C_D, DOB, Age) &
+  leq(Age,17).
+
+exception(C,P):-
```
[the remark this answers is not in the document]
Comment (labhinav) [reply claims_analysis/cardinal_care/cardinal_care.hrf]: removed

File: claims_analysis/cardinal_care/cardinal_care.hrf [hunk 3]
```diff
@@ -36,6 +36,74 @@ valid_hospitalization(C,P):-
...
+
+get_timestamp_from_date(DATE,STAMP) :-
+  evaluate(parsedate(DATE),[D,M,Y]) &
+  evaluate(maketimestamp(Y,M,D,0,0,0),STAMP)
+
+definition(parsedate(DATE),map(readstring,tail(matches(stringify(DATE),"(..)_(..)_(....)"))))
+definition(parsetime(TIME),map(readstring,tail(matches(stringify(TIME),"(..)_(..)"))))
+definition(tail(X!L),L)
+
+get_age(C_D, DOB, Age) :-
```
[the remark this answers is not in the document]
Comment (labhinav) [reply claims_analysis/cardinal_care/cardinal_care.hrf]: fixed

File: claims_analysis/cardinal_care/all_test_case_data.hdf [hunk 4]
```diff
@@ -1,8 +1,8 @@
-%% expected - covered(X) - claim1,claim3,claim5,claim6,claim8
+%% expected - covered(X) - claim1,claim3,claim5,claim6,claim8, claim_a1, claim_a2, claim_a3
 
 %% test cases for preventive_care
 
-%% expected - covered(claim1) - yes
+%% expected - covered(claim1) - yesa
```
[the remark this answers is not in the document]
Comment (labhinav) [reply claims_analysis/cardinal_care/all_test_case_data.hdf]: fixed

File: claims_analysis/cardinal_care/cardinal_care.hrf [hunk 5]
```diff
@@ -36,6 +36,74 @@ valid_hospitalization(C,P):-
...
 
+%% seperate rule for emergency room as out of network hospitals are covered
+valid_hospitalization(C,P):-
+  claim.hospitalization(C,H) &
+  hospitalization.reason(H,emergency_room) &
+  hospitalization.type_of_care(H,emergency) &
+  ~exception(C,P).
+
+
+get_timestamp_from_datetime(DATE,TIME,STAMP) :-
```
[the remark this answers is not in the document]
Comment (labhinav) [reply claims_analysis/cardinal_care/cardinal_care.hrf]: removed

File: claims_analysis/cardinal_care/schema.txt
[the remark this answers is not in the document]
Comment (labhinav) [reply claims_analysis/cardinal_care/schema.txt:1]: fixed

**Comment by labhinav** [comment]
@harshitgupta412 can you please review the new changes.

Also covered(claim12) seems to be failing, can you PTAL

**Review by harshitgupta412 — APPROVED** [review]

**Pull request merged**